plugins/performance-lab/includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -147,10 +147,11 @@ function perflab_aao_query_autoloaded_options(): array {
 	$large_options = array();
 
 	foreach ( $all_options as $option_name => $option_value ) {
-		if ( strlen( $option_value ) > $option_threshold ) {
+		$serialized_value = maybe_serialize( $option_value );
```

Comment:
Following the pattern of #1238, I think this should be:

```suggestion
		if ( is_array( $option_value ) || is_object( $option_value ) ) {
			$option_value = serialize( $option_value ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.serialize_serialize
		}
```